src/main/java/com/bravo/user/dao/model/Auth.java

```diff
@@ -0,0 +1,33 @@
+package com.bravo.user.dao.model;
+
+import com.bravo.user.enumerator.Role;
+import lombok.*;
+
+import javax.persistence.*;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "auth")
+public class Auth {
+
+  @Id
+  @Column
+  private String id;
+
+  @Column(nullable = false)
+  private String email;
+
+  @Column(nullable = false)
+  private String password;
+
+  @Enumerated(EnumType.STRING)
+  @Column
+  private Role role;
+
+  @OneToOne(cascade = CascadeType.ALL)
```

Comment:
Like that you added a CascadeType.  Is CascadeType.PERSIST necessary in this case?

Reply:
When I added it with ALL, I was thinking their might be use cases for CRUD functionality driven from the auth level. For example an admin, or an import job used to create auth users with default passwords.  I also added the constraint at the db level as seen in the screenshot, but depending on the use cause the CascadeType could be changed.
<img width="626" alt="table-constraint" src="https://user-images.githubusercontent.com/27737590/131398692-0c07c1dd-4aae-46f1-b746-fe7f1323d09f.png">